mobile/Playground.playground/Contents.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
```

Comment:
Would you mind if we left out the `mobile/Playground.playground` folder from the repo? (since the playgrounds aren't a part of our actual app)

Reply:
Good call.